release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1529,6 +1529,16 @@ In the following tables, features are marked with the following statuses:
 |Technology Preview
 |Technology Preview
 
+|Cloud controller manager for IBM Cloud
```

Comment:
Confused why this is being added to TP table?

Reply:
I think we keep them for three releases (so would go off this table next time). I might be misremembering though, I will ask Joe :)

Reply:
you were right, updated. thanks!